flutter/example/lib/main.dart

```diff
@@ -113,6 +113,12 @@ class MainScaffold extends StatelessWidget {
               child: const Text('Dart: Web request'),
               onPressed: () => makeWebRequest(context),
             ),
+            RaisedButton(
+              child: const Text('Record print() as breadcrumb'),
+              onPressed: () {
+                print('Foo bar');
```

Comment:
maybe we also Sentry.captureMessage('a message with a foo ar breadcrumb') so we can check on Sentry directly